ci/pipeline.yml

```
@@ -323,76 +322,74 @@ jobs:
-              bosh-ci: bosh
-            params:
-              BOSH_DIRECTOR_RELEASE_PATH: "bosh-release/release.tgz"
-              BOSH_DIRECTOR_RELEASE_VERSION: "latest"
-              DNS_RELEASE_FILE_PATH: "dns-release"
-              DNS_RELEASE_VERSION: "create"
-              FOCUS_SPEC: "BoshDns"
-              DIRECTOR_STEMCELL_OS: ubuntu-jammy
-              STEMCELL_OS: ubuntu-jammy
+      - in_parallel:
+          - get: bosh-dns-release
+            passed: [ pre-integration-fan-in ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)